api/circle/circle.spec.js

```diff
@@ -0,0 +1,55 @@
+require('chai').should();
+
+const app = require('../../app');
+
+const expect = require('chai').expect;
+require('chai').should();
+
+const request = require('supertest');
+
+
+const circleDAO = require('../../dao/circle');
+
+describe('/circle api', () => {
```

Comment:
Avoid using Arrow Functions in describe() and it():
http://remarkablemark.org/blog/2016/08/20/arrow-functions-in-mocha/